artemis-commons/src/main/java/org/apache/activemq/artemis/utils/collections/TypedProperties.java

```diff
@@ -329,7 +331,9 @@ public boolean containsProperty(final SimpleString key) {
       }
    }
 
-   public synchronized void decode(final ByteBuf buffer) {
```

Comment:
should keep method back compatible.

Reply:
It is back-compatible

Reply:
its not, you removed a public method, from a very common class thats exposed to clients.

Reply:
I'm missing something for sure: I've added a method using the interners, but not removed the old one...or not?

Reply:
according to diff its showing red, which means its removed.

Reply:
It is a github fault, but probably dependent by how i've formatted the code :P
If you look at the real code you will see that the method is present and used :+1:

Reply:
Ah spotted it, line 432....almost 100 line jump!

Reply:
Argh! My fault...github diff can't expect 100 line shift I think!

Reply:
haha